docs/entities/registeredservice.md

```diff
@@ -0,0 +1,49 @@
+remove the claim and the registered service is put back in the pool of available services.
+
+## Specification
+
+The definition of a RegisteredService can be obtained directly from our [RegisteredService CRD](../../config/crd/bases/primaza.io_registeredservices.yaml). The specification contains two required properties:
+
+- ServiceClassIdentity: A set of key/value pairs that identify the service class. Examples of service class identity keys include type of service, and provider of service. This property is required.
+- ServiceEndpointDefinition: A set of key/value pairs that provides two pieces of information: the service connectivity and
+the service authentication. Values can be either a string or a reference to a secret field. This property is required.
+
+A RegisteredService also has three optional properties, which gives the user more control over the resource:
+
+- Constraints: Restrictions of the registered service. A constrain could be a specific environment where the
+service cannot be claimed from. As an example, a constrain could describe that I don't want this service to be used on any environment except for production. This property is optional, when it is absent, it means there is not constrains.
+- HealthCheck: A mechanism to be able to verify the service is online and ready to use. One way this can be accomplished is
+by providing an image containing a client that can be run to test connectivity and authentication. This property is optional, when it is absent, it means the service will be considered available as soon as it is registered. 
+- SLA: Provides multiple levels of resiliency, scalability, fault tolerance and security. This allows claims to take into account the robustness of service. This property is optional, when it is absent, it means that there is no distinctions between services given the SLA.
+
+
+## Status
+
+The Status of the Service it is also defined under the [RegisteredService CRD](../../config/crd/bases/primaza.io_registeredservices.yaml). It contains only one required property to track state. The state could be one of the following: registered, available, claimed, unreachable or undetermined. The registered service state is "registered" right after creation. In most cases this will change immediately after the controller gets notified of existence of new resource, unless controller fails even before it tries to run health check. If the health check passes or is not defined, the state will change to "available". Once a RegisteredService is claimed, its state moves to "claimed". On the other hand, if the health check comes back as a failure, the state would change to "unreachable". In some cases, the health check could failed before it can determine where it fails of passes. In those cases, the state of the registered service will go to "undetermined" indicating there is an issue with the health check not with the service itself.
```

Comment:
```suggestion
The Status of the Service it is also defined under the [RegisteredService CRD](../../config/crd/bases/primaza.io_registeredservices.yaml). It contains only one required property to track state. The state could be one of the following: registered, available, claimed, unreachable or undetermined. The registered service state is "registered" right after creation. In most cases this will change immediately after the controller gets notified of existence of new resource, unless controller fails even before it tries to run health check. If the health check passes or is not defined, the state will change to "available". Once a RegisteredService is claimed, its state moves to "claimed". On the other hand, if the health check comes back as a failure, the state would change to "unreachable". In some cases, the health check could fail executing, even before it can determine whether the service is healthy or not. In those cases, the state of the registered service will go to "undetermined" indicating there is an issue with the health check not with the service itself.
```

Reply:
done